doc/api_assets/style.css

```
@@ -649,6 +698,18 @@ kbd {
   }
 }
 
+.header-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.theme-toggle-btn {
+  border: none;
+  background: transparent;
+  outline: var(--brand3) dotted 2px;
```

So the button is aligned with the `<h1>` element on big screens.
```suggestion
  outline: var(--brand3) dotted 2px;
  margin: 1.5rem 0 1rem;
```